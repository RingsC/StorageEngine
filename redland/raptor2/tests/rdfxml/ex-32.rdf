<?xml version="1.0"?>
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
         xmlns:ex="http://example.org/ns/">

  <!-- Test property attribute with rdf:resource -->

  <rdf:Description rdf:about="http://example.org/node1">
    <ex:prop1 rdf:resource="http://example.org/node2" ex:prop2="literal" />
  </rdf:Description>
</rdf:RDF>
