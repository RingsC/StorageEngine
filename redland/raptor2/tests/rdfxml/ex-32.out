<http://example.org/node2> <http://example.org/ns/prop2> "literal" .
<http://example.org/node1> <http://example.org/ns/prop1> <http://example.org/node2> .
